### PR TITLE
community/py-lz4: update to 0.9.0

### PR DESCRIPTION
**Pull request by itoffshore**
community/py-lz4: update to `0.9.0`

### CODE REVIEW

File: community/py-lz4/APKBUILD [hunk 1]
```diff
@@ -44,12 +46,17 @@ _py3() {
...
 
 	mkdir -p "$subpkgdir"/usr/share/$pkgname
 	mv "$builddir"/tests "$subpkgdir"/usr/share/$pkgname
 }
 
-md5sums="50dc6c2cb99c43ce6ed95cf6ae965f0e  lz4-0.8.2.tar.gz"
-sha256sums="6bf49061d73d69c453e892ace4586b99ccffc7de558f921d18b9418235692ac7  lz4-0.8.2.tar.gz"
-sha512sums="e516d5c3309f214fa422b040a0aacdda079395624dd4f5548ed7af637b1594012aac75b309cef260b41c10a623410f815fc089b54ca5d80955d5f2ead12841b1  lz4-0.8.2.tar.gz"
+check() {
+	echo "*** running python2 tests ***"
```
Comment (jirutka) [inline community/py-lz4/APKBUILD]: Please use `msg` (function provided by abuild) instead of `echo`.

Comment (itoffshore) [inline community/py-lz4/APKBUILD]: fixed to `msg` & rebased

**Comment by itoffshore** [comment]
fixed & rebased

**Comment by jirutka** [comment]
This package requires `nose` for tests. Now it’s automatically installed when running `setup.py test`, that’s not ideal. Please add `checkdepends="py-nose"`.

**Comment by itoffshore** [comment]
I've added the `checkdepends="py-nose"`  - however the test suite still downloads py-nose

I'll open another issue with the author

**Comment by jirutka** [comment]
I don’t think that it’s author’s fault, but the way how `setup.py test` works. :/

**Comment by jirutka** [comment]
Maybe run `nose` directly instead of via `setup.py test`?

**Comment by itoffshore** [comment]
the `python2` tests are run will the repo version of `nose` - I need to create a `python3` version of `nose` so the `python3` tests are also run with the Alpine repo version of `nose`

**Pull request closed without merging**